docs/tutorials/development-setup.mdx

```diff
@@ -0,0 +1,239 @@
+---
+title: Development Setup
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Getting Started
```

Comment:
Not sure exactly how much repeating there is with our docs on the FxA front page (I'm trying to just skim this PR) but we should probably at least cross-link the link over there